common/scripted_triggers/wc_his_character_triggers.txt

```diff
@@ -182,9 +234,18 @@ is_any_historical_character_trigger = {
 		is_garona_trigger = yes
 		is_orgrim_trigger = yes
 		is_kilrogg_trigger = yes
+		is_drekthar_trigger = yes
+		is_draka_trigger = yes
+		is_nazgrel_trigger = yes
 		
 		# Dwarves
 		is_magni_trigger = yes
+		is_muradin_trigger = yes
+		is_brann_trigger = yes
```

Comment:
I would use a bigger beardfor Brann.
I know his beard in WoW and on arts is 2-tailed and we don't have something similar, but I guess it's better to pick the long 1-tailed beard.

Reply:
I definitely considered it, but I wanted to avoid having all three Bronzebeards having the exact same beards and chose to prioritize his curled mustache as I felt it more indicative of the spirit of the character than the braids in his beard.

Reply:
> chose to prioritize his curled mustache

That's why I feel this beard is better for Brann.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120908688-c861db00-c67d-11eb-9419-c115418a2d5e.png)

</details>

Brann has big mustaches.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120908692-d4e63380-c67d-11eb-82e7-5a39051c908c.png)

</details>